web/forms.py

```
@@ -1787,3 +1795,159 @@ class StudyGroupForm(forms.ModelForm):
+        max_value=5,
+    )
+
+    comments = forms.CharField(
+        widget=TailwindTextarea(attrs={"rows": 3, "placeholder": "Any additional comments or suggestions?"}),
+        required=False,
+        label="Additional Comments (optional)",
+    )
+
+    class Meta:
+        model = SessionSurvey
+        fields = ["content_rating", "teaching_rating", "pace_rating", "materials_rating", "comments"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)